fastf1/_api.py

```diff
@@ -786,7 +788,7 @@ def timing_app_data(path, response=None, livedata=None):
         livedata: An instance of :class:`fastf1.livetiming.data.LiveTimingData` to use as a source instead of the api
```

Comment:
This line is over the character limit but breaking it into two lines caused a doc build failure in the last commit.

Reply:
You'd need additional indentation after the line break, but it should be possible. Doesn't matter here though, because it's unchanged and line lenght is only checked on the diff. So it's not a problem?

Reply:
IMO, this is good to merge? I'm not sur, what your actual open question here is or was?

Reply:
Yes I wrote it down as an aside for myself. Ready to merge if you approve